CHANGELOG.md

```diff
@@ -14,7 +14,10 @@
   * Add `authChallenge` property to `CardVaultResult`
   * Add `CardAuthChallenge` type
 * FraudDetection
-  * Bump Magnes dependency to version 5.4.0
+  * Bump Magnes version to 5.5.0
+  * Create `PayPalDataCollectorRequest`
```

Comment:
Should we note that this fixes play store rejections similar to BT CHANGELOG?

Reply:
I added a generic "Fixes Google Play Store Rejection" to the CHANGELOG. If there's a docs link to add in the near future we can always go back and add it, not sure if we want to hold off on merging until then.